src/internal/carriers/rates/rate.ts

```diff
@@ -18,7 +18,7 @@ export class Rate {
       isTrackable: Joi.boolean(),
       charges: Joi.array().min(1).items(Charge[_internal].schema).required(),
       notes: Note[_internal].notesSchema,
-      packages: Joi.array().min(1).items(RatePackage[_internal].schema).required(),
+      packages: Joi.array().items(RatePackage[_internal].schema).optional(),
```

Comment:
Everything in Joi is optional by default, so `.optional()` is redundant.  It doesn't hurt anything though.